app/src/main/res/layout/calendar_month_view.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <ImageButton
+            android:id="@+id/monthBackButton"
+            android:layout_width="wrap_content"
+            android:layout_height="30dp"
```

Comment:
```suggestion
            android:layout_height="32dp"
```
https://support.google.com/accessibility/android/answer/7101858

Reply:
The height should be even greater according to this website (https://support.google.com/accessibility/android/answer/7101858), shouldn't it?